src/test/java/edu/harvard/iq/dataverse/api/imports/ImportGenericServiceBeanTest.java

```diff
@@ -1,21 +1,70 @@
 package edu.harvard.iq.dataverse.api.imports;
 
 import edu.harvard.iq.dataverse.api.dto.DatasetDTO;
+import edu.harvard.iq.dataverse.api.dto.DatasetVersionDTO;
+
+import org.apache.commons.io.FileUtils;
+import com.google.gson.Gson;
+import java.io.File;
+import java.io.IOException;
+
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.junit.jupiter.MockitoExtension;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
+import java.nio.charset.StandardCharsets;
+
 @ExtendWith(MockitoExtension.class)
 public class ImportGenericServiceBeanTest {
```

Comment:
I'll just put this comment here. API tests didn't run: https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-11049/3/testReport/

Reply:
The build my latest changes have triggered is still running. I'll keep an eye on it.

Reply:
The last Jenkins run did succeed.